platform/src/lib/envConstants.js

```diff
@@ -1,2 +1,4 @@
 export const LEGACY_PLATFORM_ENV = process.env.NEXT_PUBLIC_ORIGINAL_PLATFORM;
 export const NEXT_PUBLIC_AUTHORISATION = process.env.NEXT_PUBLIC_AUTHORISATION;
+export const NEXT_PUBLIC_BASE_DEVICE_MONITORING_URL =
+  process.env.NEXT_PUBLIC_BASE_DEVICE_MONITORING_URL;
```

Comment:
I am assuming you've already added/updated the value of `NEXT_PUBLIC_BASE_DEVICE_MONITORING_URL` in the staging secret on secret manager. Kindly share with me it's prod equivalent so I can do the same there.

Reply:
Ok. Thanks.